libraries/testing/contracts/eosio.bios/eosio.bios.hpp

```diff
@@ -125,32 +125,32 @@ namespace eosiobios {
                                      (schedule_version)(new_producers))
    };
 
-   /**
-    * finalizer_authority
-    *
-    * The public bls key of the hotstuff finalizer in Affine little-endian form.
-    */
    struct finalizer_authority {
-      std::string              description;
-      uint64_t                 fweight = 0; // weight that this finalizer's vote has for meeting fthreshold
-      std::vector<uint8_t>     public_key_g1_affine_le; // size 96, CDT/abi_serializer has issues with std::array
+      std::string   description;
+      uint64_t      weight = 0;  // weight that this finalizer's vote has for meeting threshold
+      std::string   public_key;  // public key of the finalizer in base64 format
+      std::string   pop;         // proof of possession of private key in base64 format
```

Comment:
we didn't add the `pop` to `abi_finalizer_authority` in `bios::setfinalizer`

Reply:
We don't do pop verification inside the host function.

Reply:
I know that was by design, but why don't we do pop verification in the host function? @arhag

Reply:
This was answered today in the meeting, so I think we can resolve this conversation now.